### ISSUE TITLE
Finish editing and save a cell value on blur

### ISSUE THREAD

**Issue by SanjaySankar** [nhn/tui.grid#1145]
Is there an event that I can access which tracks when a cell of the grid has been `unfocussed`/`blurred` ? I see there is an event called `focusChange` but that just tracks changing focus from one cell to the other.
I want to ideally save the value of an editable cell when the cell is `blurred`/`unfocussed`. Eg. When the user clicks anywhere outside the component while editing is happening in a cell, is it possible to save that value as the new value on `blur` event?

**Comment by js87zz** [comment]
@SanjaySankar 
You want to trigger `blur` event when clicking **the outside of grid**? 
If it's same as your proposal, it's great idea. We will consider this event.
Thank you!

**Comment by SanjaySankar** [comment]
@js87zz Yes! Exactly. I wish to call the `finishEditing` method when the blur event gets triggered so as to save the value in the currently editing cell. Hoping it will work that way.
